File: 발표자료/ATM프로그램_2차프로젝트_Java/ATM_Java/src/ATM/main_Atm.java
package ATM;

import java.sql.*;
import java.util.Scanner;

public class main_Atm extends Bank {

    public static void main(String[] args) throws SQLException {

        String button; //초기화면 선택 버튼(switch)

        //객체 생성
        DBClass dbc = new DBClass();
        Scanner scanner = new Scanner(System.in);
        PrintfScreen myScreen = new PrintfScreen();
        Deposit myDeposit = new Deposit();
        Withdraw myWithdraw = new Withdraw();
        Remit myRemit = new Remit();
        CreateAccount myCreateAccount = new CreateAccount();
        CheckTheBalance mycheckTheBalance = new CheckTheBalance();
        ChangePassword myChangePassword = new ChangePassword();
        Administrator myadministrator = new Administrator();


        while(true)
        {
            myScreen.startScreen(); //초기화면 출력
            button = scanner.next();

            if(button.equals("1") || button.equals("2") || button.equals("3") || button.equals("4") || button.equals("5") || button.equals("6") || button.equals("7") || button.equals("8") || button.equals("cls"))
            {
                switch(button)
                {
                    case "1":
                        myDeposit.deposit();
                        break;

                    case "2":
                        myWithdraw.withdraw();
                        break;

                    case "3":
                        myRemit.remit();
                        break;

                    case "4":
                        mycheckTheBalance.check_the_balance();
                        break;

                    case "5":
                        myCreateAccount.create_acc();
                        break;

                    case "6":
                        myChangePassword.change_password();
                        break;

                    case "7":
                        myadministrator.administrator();
                        break;

                    case "8":
                        System.out.println("프로그램을 종료합니다.");
                        return;

                    //히든키(콘솔창 초기화)
                    case "cls":
                        System.out.println("cls");
                        break;
                }
            }
            else
            {
                System.out.println("\n\nError code1: You have entered an invalid value.");
                System.out.println("잘못 입력하셨습니다. 정확한 값을 입력해주세요!\n\n");
            }
        }

    }
}

File: 발표자료/ATM프로그램_3차프로젝트_Python/ATM_Python/ATM/main_ATM.py
import PrintScreen
import Deposit
import Withdraw
import Remit
import CheckTheBalance
import CreateAccount
import ChangePassword
import Administrator

while True:

    PrintScreen.start_screen()
    button = input("원하시는 업무를 선택해주세요 : ")

    if button == "1":
        print("\n1번[입금]을 선택하셨습니다.")
        Deposit.deposit()

    elif button == "2":
        print("\n2번[출금]을 선택하셨습니다.")
        Withdraw.withdraw()

    elif button == "3":
        print("\n3번[송금]을 선택하셨습니다.")
        Remit.remit()

    elif button == "4":
        print("\n4번[잔액조회]을 선택하셨습니다.")
        CheckTheBalance.check_the_balance()

    elif button == "5":
        print("\n5번[계좌 생성]을 선택하셨습니다.")
        CreateAccount.create_acc()

    elif button == "6":
        print("\n6번[비밀번호 변경]을 선택하셨습니다.")
        ChangePassword.change_password()

    elif button == "7":
        print("\n7번[관리자]을 선택하셨습니다.")
        Administrator.administrator()

    elif button == "8":
        print("프로그램을 종료합니다.")
        break

    elif button == "cls":
        print("clear")

    else:
        print("잘못 입력하셨습니다.")